### ISSUE TITLE
Problem starting Xpra on MacOS "gi.RepositoryError: Typelib file for namespace 'HarfBuzz', version '0.0' not found"

### ISSUE THREAD

**Issue by baracudaz** [Xpra-org/xpra#3221]
Freshly installed Xpra v4.2.1 is not starting on MacOS (x86_64 Intel) and throws the following error:
```python
$ Xpra
xpra main error:
Traceback (most recent call last):
  File "/Applications/Xpra.app/Contents/Resources/lib/python/gi/importer.py", line 137, in load_module
    introspection_module = get_introspection_module(namespace)
  File "/Applications/Xpra.app/Contents/Resources/lib/python/gi/module.py", line 257, in get_introspection_module
    module = IntrospectionModule(namespace, version)
  File "/Applications/Xpra.app/Contents/Resources/lib/python/gi/module.py", line 109, in __init__
    repository.require(namespace, version)
gi.RepositoryError: Typelib file for namespace 'HarfBuzz', version '0.0' not found

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Applications/Xpra.app/Contents/Resources/lib/python/xpra/scripts/main.py", line 120, in main
    defaults = make_defaults_struct()
  File "/Applications/Xpra.app/Contents/Resources/lib/python/xpra/scripts/config.py", line 1257, in make_defaults_struct
    return do_make_defaults_struct(extras_defaults or {}, extras_types or {}, extras_validation or {}, username, uid, gid)
  File "/Applications/Xpra.app/Contents/Resources/lib/python/xpra/scripts/config.py", line 1263, in do_make_defaults_struct
    defaults = read_xpra_defaults(username, uid, gid)
  File "/Applications/Xpra.app/Contents/Resources/lib/python/xpra/scripts/config.py", line 412, in read_xpra_defaults
    dirs = get_xpra_defaults_dirs(username, uid, gid)
  File "/Applications/Xpra.app/Contents/Resources/lib/python/xpra/scripts/config.py", line 433, in get_xpra_defaults_dirs
    dirs = get_default_conf_dirs() + get_system_conf_dirs() + get_user_conf_dirs(uid)
  File "/Applications/Xpra.app/Contents/Resources/lib/python/xpra/platform/paths.py", line 65, in get_default_conf_dirs
    return envaslist_or_delegate("XPRA_DEFAULT_CONF_DIRS", do_get_default_conf_dirs)
  File "/Applications/Xpra.app/Contents/Resources/lib/python/xpra/platform/paths.py", line 27, in envaslist_or_delegate
    return impl(*args)
  File "/Applications/Xpra.app/Contents/Resources/lib/python/xpra/platform/darwin/paths.py", line 75, in do_get_default_conf_dirs
    return [os.path.join(get_resources_dir(), "etc", "xpra")]
  File "/Applications/Xpra.app/Contents/Resources/lib/python/xpra/platform/paths.py", line 183, in get_resources_dir
    return env_or_delegate("XPRA_RESOURCES_DIR", do_get_resources_dir)
  File "/Applications/Xpra.app/Contents/Resources/lib/python/xpra/platform/paths.py", line 32, in env_or_delegate
    return impl(*args)
  File "/Applications/Xpra.app/Contents/Resources/lib/python/xpra/platform/darwin/paths.py", line 27, in do_get_resources_dir
    macapp = get_OSXApplication()
  File "/Applications/Xpra.app/Contents/Resources/lib/python/xpra/platform/darwin/__init__.py", line 44, in get_OSXApplication
    from gi.repository import GtkosxApplication #@UnresolvedImport
  File "/Applications/Xpra.app/Contents/Resources/lib/python/gi/importer.py", line 139, in load_module
    raise ImportError(e)
ImportError: Typelib file for namespace 'HarfBuzz', version '0.0' not found
```

Xpra has been installed via `brew install xpra` however the xpra package downloaded for the website has the same issue:
```
$ brew info xpra
xpra: 4.2.1,0
https://www.xpra.org/
/usr/local/Caskroom/xpra/4.2.1,0 (64.7MB)
From: https://github.com/Homebrew/homebrew-cask/blob/HEAD/Casks/xpra.rb
==> Name
Xpra
==> Description
Screen and application forwarding system
==> Artifacts
Xpra-Python3-x86_64-4.2.1-r0.pkg (Pkg)
```

**Comment by tdietel** [comment]
I have encountered the same issue with the Xpra.pkg package downloaded directly from the website. 

I have also tested with Xpra 4.2 from the web page, and in that case I run into a different problem, but HarfBuzz seems to be ok.

**Comment by totaam** [comment]
This was fixed in 7e6a8cceb08bdf8a20f8e5a590b6c516ccacc51a, until I can produce an update 4.2.x build, please use the [beta builds](https://xpra.org/beta/MacOS/)

**Comment by totaam** [comment]
@tdietel "and in that case I run into a different problem" - please be specific, there's absolutely nothing I can do with this information.
